modules/lib/api.json

```diff
@@ -884,6 +884,14 @@
 							"type": "string",
 							"example": "docbook.odd"
 						}
+					},
+					{
+						"name": "nav-root",
+						"in": "query",
+						"schema": {
+							"type": "string",
+							"example": "/1/4"
```

Comment:
The test below uses a value of `1/4`, but this example has a leading slash - `/1/4`. Is the leading slash optional?

Reply:
You are right, it should be `/1/4` won't work, I will update it